app/qml/FeaturePanel.qml

```diff
@@ -145,8 +145,8 @@ Drawer {
             fontBtnColor: InputStyle.highlightColor
             rowHeight: InputStyle.rowHeightHeader
             titleText: featurePanel.formState === "Edit" ? qsTr("Edit Feature") : qsTr("Feature")
-            backIconVisible: false
-            backTextVisible: true
+            backIconVisible: !saveButtonText.visible
+            backTextVisible: saveButtonText.visible
```

Comment:
Maybe this is more self explanatory to me 

```suggestion
            backIconVisible: featurePanel.formState === "Edit"
            backTextVisible: featurePanel.formState !== "Edit"

```

Reply:
It is maybe a bit clearer, but the condition is a bit more complex than what is suggested:
https://github.com/lutraconsulting/input/blob/1219_fix/app/qml/FeaturePanel.qml#L158
Therefore I would keep it as it is now or make a function out of that condition, so we would have on single place